✨ Improving docs with mkdocs replacing raneto

RCristiano opened the pull request.

RCristiano committed:
✨ Improving docs with mkdocs replacing raneto

vc0sta commented on mkdocs/podman-compose.yml.j2:
```diff
@@ -0,0 +1,17 @@
+      - {{ port }}
+{% endfor %}
+    volumes:
+      - .:/docs:Z
```
Vamo botar isso numa subpasta pra seguir o mesmo padrão dos outros?

RCristiano replied:
Esse aqui eu tinha me esquecido, mas é um erro mesmo porque vai compartilhar o *-compose.yml com o container

RCristiano replied:
Resolved by [f8adce9](https://github.com/vc0sta/mediaserver-config/pull/13/commits/f8adce9facaee6a066402587f205065994fbaa20)

vc0sta commented on mkdocs/podman-compose.yml.j2:
```diff
@@ -0,0 +1,17 @@
+version: '3'
```
Faz um favor? atualiza todos os *-compose pra versão mais nova?

RCristiano replied:
Resolved by [3c6f37b](https://github.com/vc0sta/mediaserver-config/pull/13/commits/3c6f37be29f86e2a670e45df211e38c5c6efc016)

RCristiano committed:
🚚 Change standard location to docs

RCristiano committed:
🐛 Fix: remove unnecessary item from mkdocs sync

RCristiano committed:
👷 Update all compose file versions

Resolves https://github.com/vc0sta/mediaserver-config/pull/13#discussion_r997432937
